source/process/documentation-UItext-guidelines.rst

```diff
@@ -0,0 +1,193 @@
+User Interface Text Guidelines
+===============================
+This is the Mattermost style guide for text used in the Mattermost product. Use these guidelines to ensure the the text on the Mattermost user interface (UI) is consistent and clear.  These guidelines cover two types of UI text:
+
+- The messages that are triggered because something happened
+- The static text on the user interface
+
+.. note::
+
+Contributions that do not adhere to this style guide will receive edit requests; we'll merge your contribution once you edit the text to conform to these guidelines. You might want to look at the `UX design guidelines`_ as well.
+
+....
+
+.. contents:: 
+   :depth: 2
+
+....
+
+General principles for in-product text
+---------------------------------------
+
+- Keep in mind that your users are from all over the world. Use words that are clear and easy to understand.
+- Don't blame the user. Inform them about what happened, explain why it happened, and suggest a way forward.
+- Try to be concise.
+- Try to use the active voice, except in these cases:
+   - If you'll end up blaming the user. For example, don't say *You entered an incorrect password*. Instead, say *The password is incorrect*.
+   - If you're describing what just happened. For example, it's okay to say *Your incoming webhook has been set up*.
+   - If the subject (the doer of an action) is the Mattermost application itself. For example, it's okay to say *The image was deleted* instead of *The server deleted the image*.
+- Use the past tense to say what's already happened, the present tense to describe a current state or condition, and the future tense to state something that is very definitely going to happen.
```

Comment:
0/5 and I'll let other PMs chime in but we have been trying to avoid the past tense in most cases. ie "The image was deleted" or "Your incoming webhook has been set up" can potentially be cleaned up by saying "The image is deleted" or "Your incoming webhook is set up" and still be accurate.

Reply:
+1 for using the present tense - it'll keep the text simpler and more direct. It is also consistent with point 7 of [blog content guidelines](https://docs.mattermost.com/process/marketing-guidelines.html#guidelines-12-tips-for-sparkling-content) which recommends using the active voice

Reply:
Changed to the present tense.

Reply:
Thanks. In that case does this phrase still apply? 

> Use the past tense to say what's already happened

Reply:
It doesn't. I debated with myself on whether to leave it in because there *might* be rare instances when the past tense is applicable ("The server crashed"). Am removing it now.